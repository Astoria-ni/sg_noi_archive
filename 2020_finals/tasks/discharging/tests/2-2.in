1400
1637050 3102148 3121763 3260967 5578564 6545909 8318047 9567381 9631530 10180671 10303059 11074684 11201451 11251117 11517012 12263683 12383927 13131098 13357996 13901260 14031063 14623023 15205958 15287554 16566793 17036563 18642015 18947268 19576356 20014182 20261104 20384360 20779408 21357508 22249923 22763895 23112757 23342955 23470282 23829955 24135138 24383084 25699007 25893990 26348657 26597328 27147573 28089958 29240423 29435041 30131116 30724071 31322315 31380199 31759996 31860648 32658969 32698679 33749251 33957575 34470122 35160136 35347510 35522327 35753978 36648367 37059842 40038625 40199521 40214728 40721246 41041508 41191812 42284484 42334630 42427493 42624542 46404149 46933642 47496748 47661049 49776897 51293255 54570163 55162935 55884894 56917021 57322106 58065405 59649644 60210343 60443285 61427457 62441168 62808702 63858550 63988649 64962896 65123890 65187539 65283343 66460811 67756906 68454629 69062173 69448304 70428498 71065334 71970312 72845616 74333195 74467489 74592830 74772826 74791511 75095130 75712707 76077402 76586130 77654168 78020311 78903794 81687438 81726473 83782474 84060555 84547711 84630838 84693467 84830929 85241245 87372150 87808034 88819938 89623127 90981141 92203532 94523180 94642250 94685473 94720184 94917498 95344312 95707154 96337017 96562096 97822573 97949395 98319263 99003135 99504314 100348111 100397097 100960847 104114147 105504289 105843742 106256983 106641328 106709053 107118239 107466390 108108453 108414221 108736010 109029120 109302771 109481910 109680971 109929751 110348912 110593784 110720707 111093126 111739063 114234812 116065160 116217466 116561974 118724702 120436567 123713617 123971101 124406701 124810972 124915461 125108492 128143114 129006291 129123754 129233704 129337740 129926216 130029788 130297087 130839770 131136073 132098939 134068674 134510978 134783362 136104715 139070026 139745391 141163941 141460452 141599681 141766883 142043245 144210762 145892570 147959272 148101737 148474729 149135881 150420311 150478897 150729263 150944126 151608560 152301617 152396159 152782914 152963589 153455277 153532711 154944334 155084930 155479282 155679347 156098957 156772410 156830012 157722125 157910211 158216883 158557212 158643643 159021414 159276867 160461053 160550117 162082989 162274852 163012933 164157559 164219406 164670647 164970887 166116805 166940647 167002461 167567553 168032279 168857753 168898138 169607209 171674348 172509301 173059985 173126496 173713527 174420060 174534684 174744897 174865158 175646523 177452111 179981342 180709055 182908705 183669910 184297101 184767210 185342273 185889167 188613470 188747412 189274782 190588898 191590686 191764072 191936974 194438719 195013503 198025700 198365881 200234435 201868400 205186394 205267116 206790017 207755056 207973932 208438170 208758425 209274861 209599197 210231309 210532709 211103844 211137873 211331131 211492242 211687397 211842437 213352029 214693150 215012489 215268541 217087251 217312550 217343273 217495314 217678374 218001653 218498591 218671039 218852530 220210639 220908387 221617827 222537850 222868783 222944039 223203924 223303494 223903917 223955903 224600374 225091031 227324773 227987417 229615504 229713907 229862528 230319367 230741312 230789592 231111783 232663064 232801844 233908544 234545333 234574154 234686809 234984888 234990458 235094594 237547712 238469783 240175973 241794484 242104102 242434430 242920195 244084145 244439709 245976872 246113775 246797792 246940968 246983279 246996205 248513191 249584714 249708313 249816098 250652399 250828139 250874119 251194563 252448874 253906438 254435337 255873560 256029866 256895577 256947073 257784272 259898653 260268962 260387462 261167452 261868441 263056509 265081302 267274671 267325846 267585490 268916567 270279092 270417179 270608693 270651344 272453670 272625815 273132254 274522799 274600723 274621208 275672855 276155331 276283527 277125803 277160739 277195781 277273231 278015787 279382917 279872461 280119529 281249087 282309303 283190461 284692046 284722570 284892115 285742659 285764659 286130141 287022769 287244369 287361336 288220123 288699438 289043643 290490327 291616210 292605932 293796941 293941269 294004463 294200649 294642942 295375515 296047698 296517968 297449941 298590853 300744272 303919351 304077543 305612270 306581288 307749108 308195589 308547870 309204527 309938718 312269395 312588517 314172070 314714605 316295282 316932166 317009314 318120028 318742200 319491490 319543408 321186464 321197374 321262949 321374965 321733381 321869710 322157278 322190778 322515192 323307432 323942080 324155267 324664362 325182507 326891618 327076471 327528846 327802918 328401927 328462434 328543998 328783111 328983108 329087916 329103528 330485386 331670431 331727821 332438861 332650171 333383704 335168656 335699087 338233966 338560119 339138199 339373581 339501804 341218146 341361499 341803673 343775116 344014223 344761833 346354630 346811185 347362388 347624406 349259841 349263003 349354183 349845556 350060149 350471368 351222834 352528364 352948148 353355037 354672540 354884385 356783074 357773327 358180138 358910091 359326137 359613804 361297937 361793487 362037445 362328200 362785518 362809543 362885269 364953596 366249813 366523354 367025946 367091342 367283629 367401845 367682361 368749992 368786120 368966456 369829696 369847381 372865944 372892899 373073990 373951404 374216611 374797576 375396679 375866968 376324123 377539848 377698499 378727897 379687168 380012344 380071193 380562015 381483948 381612064 381647334 381965005 382291267 382363474 382777724 383572269 384072993 384666259 384821999 384871385 386077689 386236683 386459241 386992895 387987403 388233385 388731782 389966867 392418060 392464689 392596921 392811506 392914640 393795819 393843717 395013817 395053209 397492229 397673287 399264093 399910307 400449991 401337190 401397733 401684989 401746416 401752437 402919919 403312989 403596072 403686553 403993020 404310269 405723012 406422757 408048884 408704531 409096317 409131221 409507676 412278144 412874711 413357365 415796453 416013981 416876756 417563227 418121336 418133532 418439418 418535691 422209883 423615938 424194913 424855619 426047962 429707796 431346572 431938813 432049036 432217057 433281616 433936450 435088225 436498373 436730237 437317144 437874851 438437652 438496529 438730177 438972615 441090767 442954263 444366166 445363891 445505788 445841162 446415735 446894611 447106468 447234512 447305970 447651887 447733607 448535899 449251031 449542187 449905758 450144619 450680504 451154277 452069489 452351803 454532587 455672713 457165670 457380662 457468145 458015589 459441543 459980624 460155299 460454736 461363740 462270638 462382183 462467224 463041688 463829192 463957705 463959727 464021143 466226140 468065038 468325047 468537811 468566504 468813155 469359402 469366006 470011587 474076351 474507373 474732485 474785210 476630810 478561423 479482524 480703218 481610745 483550083 483848228 484489846 484884866 485785042 486169974 486305856 486642489 488209205 488822131 490111841 490682047 490975407 491226365 492127213 492912699 493588498 494347307 495031303 496068478 496327567 496454938 497878693 501274857 502900617 503261592 504362018 504480130 504521816 506848903 507070015 507852488 507972405 508553369 509454654 509511367 510448072 510571062 511197220 511460329 511587104 512524504 512670214 512800154 513362079 515468746 516109233 516397494 517056215 519280345 519283389 519881345 521522289 521525858 524198918 525057586 525524049 525714869 526774141 527322452 528796225 529451286 530673313 532125498 533150653 534149956 535136586 535191063 535412984 535421365 536076122 538627953 538885540 539151057 540317131 541617907 541862945 542162884 543058756 543208364 543515822 543687653 544495598 544873078 545018495 546253176 548020327 548580076 548863327 549308842 549992682 550817683 551986590 552603188 553324492 554125573 554621366 556674258 557195314 557304356 557983029 558989211 559170669 560579453 561114629 562641307 563045709 563328769 563646643 564720478 564909290 565471214 566225726 567099345 569529837 570251720 571623910 572006146 572944336 573055118 573698294 575123021 575188435 577757037 578902804 579529695 579590550 579819237 580155196 581044898 581949175 582313754 582831722 583015586 585971234 586440386 587313660 588780231 589418459 589645680 589701024 589729083 589804913 591309980 592139491 592489963 592679627 592797549 593246282 593990072 594632738 595664840 597405195 597450505 598526568 598884772 601004903 601514868 601966813 602279523 603222983 603483303 604426060 605106590 605890921 607669417 607717170 608107947 608660788 609119519 609478378 609640632 610712624 611523664 611606981 611959889 612838743 614097525 615922797 616862074 617166627 618084071 618223668 618315996 618547808 618858987 619661972 621120224 623303766 626446265 627404649 627480941 627994504 628308317 629898754 630875040 633472524 635877719 636089570 637287424 639136140 639251909 640177393 640398940 641581691 642199403 642237955 642424418 642914861 644696358 644749019 644953525 649975097 651494057 652797771 652897834 652903966 654916156 655095768 655482989 656205185 657670286 658220978 659078486 659499463 660078360 661103714 662259967 663715424 665045760 665375624 665485007 665593752 666100510 666209931 667407996 670199464 670304413 670320860 670559908 670676310 670710393 671074868 671081342 672021436 672197287 672652507 672761041 673658978 673926628 674871251 675172956 676853065 678212244 679187766 679618410 679954153 680448393 680886099 680907783 681572613 682774411 683375847 683751776 684244992 684416569 684445526 685086423 686845801 687038978 687231528 687532818 688181878 690999412 691275044 691601406 691643601 692194469 692324067 692325769 692650299 693148097 694894942 694913697 694937619 695544391 695602773 695836282 695934502 695983015 696345795 696389421 696682614 697325738 697398173 697532448 697715269 697768416 698154113 698421894 699304712 700946691 701500669 702587138 702858631 702907346 703007092 703307094 706552723 706925860 707158705 709267467 709280475 711917266 713547747 714937047 715089770 715789790 718407877 718512737 718627266 719618556 720256168 721307801 721970147 723864657 724699382 724902370 725244791 726743314 727666915 728346479 728354260 728666554 728701054 728827846 729323588 730137976 731728853 732590301 732914072 733231385 735241604 735682073 736821631 737827793 738009419 741209705 741857018 742134322 743373303 743431723 744151078 744196360 745700037 746064068 746389335 746507086 747154184 747846018 748075442 748126770 749362162 749514613 750282334 750359165 750411893 751042875 751328197 753647877 754134152 754869739 755814823 756024405 757138011 757178460 757503715 757742319 758346102 758515679 758708170 759536473 759596022 760472147 761839883 762419343 762986041 764637402 765801510 767368091 767787161 769508790 769540078 769566177 771513486 772905358 773001596 773638323 777159722 778236012 779211602 779747947 780024660 782717242 782990126 783896364 784405540 785016673 785054955 785374350 787482020 788650318 789167999 789666894 791375289 792509272 792573649 792684991 792697947 793194312 793529206 794655421 795105515 795716788 796493375 796518434 796752913 798251156 798406224 798472513 801515668 802058520 804509876 805586309 805801237 806787877 807465238 807685457 807965597 808927363 809398818 809975932 810038253 811141759 811479464 811872979 812851922 813659423 813920510 813963003 815551239 817142957 817750524 817863961 818647092 821057162 821172847 822247609 822721701 823092013 823427133 823771182 824525151 824568148 825401756 825467727 826056946 826593875 826796438 827141906 828524710 828976492 829403087 829556625 830828593 830857635 832237711 833177445 833541691 833655517 834139850 834380157 836193877 838433029 838844031 842088976 845823215 845838296 846176660 846202022 847205844 847269307 847756163 848299749 849845375 851533372 851983660 852147596 854505917 856750801 857067517 857447187 858088647 858155649 861757021 862145735 864210214 864799439 865361403 865501651 866940372 867562916 868558776 869774454 869910499 870167515 871210314 873206080 873830650 874152306 876082937 876124747 876571317 876999631 877313861 877328489 878578971 878617853 880020130 881438712 882760840 882809145 882816456 882949253 883057125 883248832 883825026 884650103 886102114 886967498 887100350 889009897 890285835 890964020 892762996 893123644 894799745 897044356 897341453 897976205 898611091 898775464 899433031 899438483 899686110 900198802 903708011 904333324 904999327 905014504 905255624 905619719 907986291 908411284 909336203 909779417 910003489 910789552 910997741 911117908 911124677 911225261 911511558 912767286 913101268 913818979 914676070 915380227 915425281 915638269 916766170 916910196 917286573 917304116 919744862 920273860 920365798 921328660 921704441 921782085 923350471 924129077 924549930 925563986 925581504 925764246 925846156 926574087 926839353 926990910 928371450 929977536 930213553 930551634 931279389 933115835 933721764 934007295 934339069 935924629 938219624 938587718 938824552 938935802 940376376 942381909 942549340 942748889 944150599 944632211 944757073 945163057 945972154 946034532 947001331 947246948 947828935 949221723 950749011 950932309 951381292 951524277 954093881 954310546 954388829 954390217 954555144 954880743 957934027 958778619 960509511 960588423 961007339 962232771 963642322 963665670 964049547 964240956 964619558 965714398 966137344 966579485 967610157 967799073 967921080 967935723 968132252 969154451 969849582 970130150 970240459 972441350 973352544 974315684 975333186 975550505 975872283 975978828 976721748 977839386 978504596 981288374 981346085 983092878 983427312 983545803 983837842 983947217 984317649 986029891 988061288 988102809 988294018 989342115 991551971 991662139 992725107 992895047 992974154 993300232 994433670 994703927 995433854 997570233
