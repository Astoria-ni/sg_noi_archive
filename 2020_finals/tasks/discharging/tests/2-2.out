1355415559201
